core/tests/protractor_utils/forms.js

```diff
@@ -61,7 +62,7 @@ var GraphEditor = function(graphInputContainer) {
   var createEdge = async function(vertexIndex1, vertexIndex2) {
     var addEdgeButton = graphInputContainer.element(
       by.css('.protractor-test-Add-Edge-button'));
-    await addEdgeButton.click();
+    await action.click('Test Add Edge Button', addEdgeButton);
     await browser.actions().mouseMove(
       vertexElement(vertexIndex1)).perform();
```

Comment:
ditto with the `vertexElement(vertexIndex1)` and `vertexElement(vertexIndex2)`

Reply:
Done!